### DISCUSSION TITLE
Things that I like in CLion and maybe you could like too in Zed

### DISCUSSION THREAD

**Discussion by decomoraes** [zed-industries/zed#6509] [Feature Requests]
Hi guys, I'd like to share with you the things that I really enjoy in CLion and it would be nice to see the same in Zed.
1. The ability to collapse the code from bottom:

<img width="380" alt="Screen Shot 2023-03-21 at 15 14 56" src="https://user-images.githubusercontent.com/32784886/226704193-90726360-beda-4e03-b45a-e2e22be362ac.png">

2. Three edges with tools, it's very useful, for example to access the database:

<img width="1800" alt="Screen Shot 2023-03-21 at 15 18 37" src="https://user-images.githubusercontent.com/32784886/226705120-6cfb1ab9-a87a-48bb-b7c8-618f1506c830.png">

3. Regions, I really use it, it's good to let my code organized and collapsible.

<img width="376" alt="Screen Shot 2023-03-21 at 15 24 21" src="https://user-images.githubusercontent.com/32784886/226705946-276ecdf9-97b2-401b-97ce-fa9c3422bd5e.png">

5. And Dracula Theme, hehe, please!

That's it guys, I'd love to see some of these features, I'd love to use Zed as my main IDE.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
